composer.json

```diff
@@ -12,6 +12,7 @@
         }
     ],
     "autoload": {
+        "files": ["src/functions_include.php"],
```

Comment:
Ah yep.

So why the include file specifically?

Seeing as it's namespaced I feel like you can get away without the method_exists check and just add `src/functions.php` to the files section directly.